CHANGELOG.md

```diff
@@ -4,6 +4,12 @@
 
 Not yet released; provisionally v2.0.0 (may change).
 
+### Better Server Skew Behavior
+
+GetLatestSignedLogRoot will return `codes.Unavailable` when the requested
```

Comment:
nit: reword to make it clear that this only affects clients who are using the new `first_tree_size` request parameter.